src/main.rs

```diff
@@ -46,5 +46,9 @@ fn main() {
     let client_addr =
         args.flag_client_address.parse::<SocketAddr>().expect("Expect to parse client address");
 
+    println!("Starting Emerald Connector - v{}",
+             VERSION.unwrap_or("unknown"));
+    println!("Listen for connection on http://{}", addr);
+
```

Comment:
Better to use log instructions, println! always a bad idea in production code, only for debugging one two times probably